cookbook/workflow/homestead.rst

```diff
@@ -0,0 +1,66 @@
+.. index:: Vagrant, Homestead
+
+Using Symfony with Homestead/Vagrant
+====================================
+
+In order to develop a Symfony application, you might want to use a virtual
+development environment instead of the built-in server or WAMP/LAMP. Homestead_
+is an easy-to-use Vagrant_ box to get a virtual environment up and running
+quickly.
```

Comment:
I definitely think it's a good idea to link to http://www.whitewashing.de/2013/08/19/speedup_symfony2_on_vagrant_boxes.html in a tip box.

I'll also add another paragraph after this with the advantages/disadvantages of using vagrant.